src/box/tuple_compression.h

```diff
@@ -0,0 +1,53 @@
+#pragma once
+/*
+ * SPDX-License-Identifier: BSD-2-Clause
+ *
+ * Copyright 2021, Tarantool AUTHORS, please see AUTHORS file.
+ */
+
+#include "trivia/config.h"
+
+#if defined(ENABLE_TUPLE_COMPRESSION)
+# include "tuple_compression_impl.h"
+#else /* !defined(ENABLE_TUPLE_COMPRESSION) */
+
+#include "tuple.h"
+
+#if defined(__cplusplus)
+extern "C" {
+#endif
+
+static inline void
+tuple_compress_raw(struct tuple_format *format, const char *data,
```

Comment:
I don't like that compression functions are defined in memtx and are named appropriately while decompression functions are defined in engine-independent file. Let's move decompression functions to memtx as well (i.e. add `memtx_` prefix both to file names and functions). Can we group them so that they are defined in the same file?

Reply:
Fixed